cmd/netgoal/generate.go

```diff
@@ -426,19 +426,13 @@ func saveGoalTemplateToDisk(template netdeploy.NetworkTemplate, filename string)
 	return err
 }
 
-func generateWalletGenesisData(wallets, npnHosts int, totalStake int64) gen.GenesisData {
+func generateWalletGenesisData(wallets, npnHosts int) gen.GenesisData {
 	data := gen.DefaultGenesis
 	if npnHosts > 0 {
 		wallets = 2 * wallets
 	}
-	// cloud templates use percentage of stake
```

Comment:
it was my incorrect assumption after briefly looking into net template validation code in goal. This change simply reverts pct-based calculation.